SentimentAnalysis.ipynb

```diff
@@ -15,8 +19,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip3 install -r requirements.txt\n",
-    "!pip3 install thirdai --upgrade"
+    "# !pip3 install thirdai --upgrade\n",
+    "# !pip3 install datasets\n",
```

Comment:
We may need to freeze the `datasets` package version for internal tests to pass.